### PR TITLE
update to v3 api

### PR DESCRIPTION
**Pull request by cohadar**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,11 +5,11 @@ RUN go get github.com/Masterminds/glide
 WORKDIR /go/src/github.com/jirwin/burrow_exporter
 COPY . /go/src/github.com/jirwin/burrow_exporter
 RUN glide install
-RUN go build burrow-exporter.go
+RUN go build -o burrow-exporter
```
Comment (solsson) [inline Dockerfile]: This is the only difference I can see from #9. What does it do?

**Comment by cohadar** [comment]
It makes sure output file is named "burrow-exporter". It is the same as calling "go build burrow-exporter.go" only more standard

**Comment by ercliou** [comment]
One small detail:
I don't think this metric "maxoffset" exists anymore:
https://github.com/jirwin/burrow_exporter/blob/master/burrow_exporter/exporter.go#L54

I haven't used v2, not sure what this metric means, but the [Burrow wiki](https://github.com/linkedin/Burrow/wiki/http-request-consumer-group-status) doesn't mention this metric. Also I've just tested on Burrow, it's not exposing this field.

(commenting this on another PR too, not sure which will be the final one)

**Comment by jirwin** [comment]
Closing this since I merged #12. Thanks for submitting this!

**Pull request closed without merging**